db/sql/00_msar.sql

```diff
@@ -2135,16 +2135,21 @@ $$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
 
 
 CREATE OR REPLACE FUNCTION
-msar.drop_table(tab_id oid, cascade_ boolean, if_exists boolean) RETURNS text AS $$/*
```

Comment:
Rethinking about this made me realize that we could allow `if_exists` with this function if we change the datatype of tab_id from `oid` -> `regclass` and we can document in the python function's docstring that `if_exists` only works when the `tab_id` is a fully qualified table name. I'd like to know your thoughts on this @mathemancer, @seancolsen.

Reply:
At the SQL level, I prefer keeping the flag for the `schema, tab_name` version, and removing it from the `tab_oid` version where it makes no sense (without custom exception handling). For the API level, I prefer defaulting to returning a success marker (e.g., the table name) if the table actually existed and was dropped, and a `null` if nothing happened (because the table didn't exist).